src/basic.js

```diff
@@ -299,8 +299,14 @@ function ForceHLAllObjects() {
  * Delay then Force Highlight objects
  *  This is used to wait for DOM to populate before filling in problem details in left panel.
  */
-function delayForceHLAllObjects() {
-	setTimeout(ForceHLAllObjects, 100);
+function delayForceHLAllObjects(e) {
```

Comment:
I'm not sure why do we need `e`?

Reply:
Its only used for the debugging print (that is commented out), to get the event type.